missing types/career.type

The file `src/types/career.type` does not exist in [types](https://github.com/gitpod-io/website/tree/main/src/types) folder

Referenced by: [/src/components/careers/modal.svelte](https://github.com/gitpod-io/website/blob/main/src/components/careers/modal.svelte#L2) and [src/contents/careers.ts](https://github.com/gitpod-io/website/blob/main/src/contents/careers.ts#L1)

The error is also reported by [svelte-check](https://www.npmjs.com/package/svelte-check)

```shell
$ npx svelte-check

[...]
.../website/src/components/careers/modal.svelte:2:31
Error: Cannot find module '../../types/career.type' or its corresponding type declarations. (ts)
<script lang="ts">
  import type { Career } from "../../types/career.type";
  import Modal from "../modal.svelte";
[...]

====================================
svelte-check found 53 errors, 10 warnings and 70 hints
```

Along with other errors and warning

--- a/src/types/career.type.ts
+++ b/src/types/career.type.ts
@@ -3,7 +3,7 @@ type Listing = {
   items: string[];
 };
 
-export type Job = {
+export type Career = {
   title: string;
   intro: string;
   paragraphs: string;

